apps/src/panels/PanelsLabView.tsx

```diff
@@ -32,7 +32,7 @@ const PanelsLabView: React.FunctionComponent = () => {
   );
   const skipUrl = useAppSelector(state => state.lab.levelProperties?.skipUrl);
   const offerTts =
```

Comment:
I don't have full context on the rename to `offerTts` but I'm wondering whether we'd also want to rename this variable.

Reply:
Oh yeah, I can update these as well